### PR TITLE
feat: v0 shadow levels styles migration from v9 for teams and dark theme

### PR DESCRIPTION
**Pull request by GianoglioEnrico**
This Pull Request updates v0's shadow levels styles to match the v9 ones.
We are only changing the teams and dark themes shadow levels.

This is the spec we followed for the migration:
Screenshot 2022-10-24 at 11 36 55

## Related Issue(s)

Fixes #

### CODE REVIEW

File: packages/fluentui/react-northstar/src/themes/teams-dark/siteVariables.ts [hunk 1]
```diff
@@ -11,10 +11,10 @@ export const focusOuterBorderColor = colors.white;
 //
 // SHADOW LEVELS
 //
-export const shadowLevel1 = '0 .2rem .4rem -.075rem rgba(0, 0, 0, .25)';
-export const shadowLevel2 = '0 .4rem 0.7rem -0.1rem rgba(0, 0, 0, .25)';
-export const shadowLevel3 = '0 .8rem 1rem -0.2rem rgba(0, 0, 0, .25)';
-export const shadowLevel4 = '0 .6rem 1.8rem -0.4rem rgba(0, 0, 0, .25)';
+export const shadowLevel1 = '0 0 2px rgba(0,0,0,0.30), 0 2px 4px rgba(0,0,0,0.25)';
```
Comment (jurokapsiar) [inline packages/fluentui/react-northstar/src/themes/teams-dark/siteVariables.ts:14]: aren't these changes for v2 theme? do we want to modify v1 theme as well?

Comment (GianoglioEnrico) [reply packages/fluentui/react-northstar/src/themes/teams-dark/siteVariables.ts:14]: Thank you Juraj I am gonna ask this, in the task was not specified!

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/6ced976a8d0e6a0e2e207da8fe0eb810e2bd19bc'>6ced976a8d0e6a0e2e207da8fe0eb810e2bd19bc</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## Perf Analysis (`@fluentui/react-northstar`)
<details><summary>Perf tests with no regressions</summary>

Scenario | Current PR Ticks | Baseline Ticks | Ratio
:--- | ---:| ---:| ---:
PortalMinimalPerf.default | [151](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/PortalMinimalPerf.default.html) | [121](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/PortalMinimalPerf.default_base.html) | 1.25:1
AccordionMinimalPerf.default | [127](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/AccordionMinimalPerf.default.html) | [103](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/AccordionMinimalPerf.default_base.html) | 1.23:1
TreeWith60ListItems.default | [133](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/TreeWith60ListItems.default.html) | [108](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/TreeWith60ListItems.default_base.html) | 1.23:1
HeaderSlotsPerf.default | [652](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/HeaderSlotsPerf.default.html) | [540](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/HeaderSlotsPerf.default_base.html) | 1.21:1
BoxMinimalPerf.default | [287](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/BoxMinimalPerf.default.html) | [239](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/BoxMinimalPerf.default_base.html) | 1.2:1
ChatDuplicateMessagesPerf.default | [232](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ChatDuplicateMessagesPerf.default.html) | [197](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ChatDuplicateMessagesPerf.default_base.html) | 1.18:1
TextAreaMinimalPerf.default | [392](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/TextAreaMinimalPerf.default.html) | [331](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/TextAreaMinimalPerf.default_base.html) | 1.18:1
ButtonSlotsPerf.default | [453](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ButtonSlotsPerf.default.html) | [387](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ButtonSlotsPerf.default_base.html) | 1.17:1
DropdownManyItemsPerf.default | [571](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/DropdownManyItemsPerf.default.html) | [492](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/DropdownManyItemsPerf.default_base.html) | 1.16:1
ReactionMinimalPerf.default | [320](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ReactionMinimalPerf.default.html) | [275](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ReactionMinimalPerf.default_base.html) | 1.16:1
AnimationMinimalPerf.default | [429](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/AnimationMinimalPerf.default.html) | [377](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/AnimationMinimalPerf.default_base.html) | 1.14:1
HeaderMinimalPerf.default | [296](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/HeaderMinimalPerf.default.html) | [260](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/HeaderMinimalPerf.default_base.html) | 1.14:1
InputMinimalPerf.default | [992](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/InputMinimalPerf.default.html) | [872](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/InputMinimalPerf.default_base.html) | 1.14:1
SegmentMinimalPerf.default | [281](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/SegmentMinimalPerf.default.html) | [246](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/SegmentMinimalPerf.default_base.html) | 1.14:1
PopupMinimalPerf.default | [515](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/PopupMinimalPerf.default.html) | [458](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/PopupMinimalPerf.default_base.html) | 1.12:1
RadioGroupMinimalPerf.default | [373](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/RadioGroupMinimalPerf.default.html) | [339](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/RadioGroupMinimalPerf.default_base.html) | 1.1:1
TreeMinimalPerf.default | [666](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/TreeMinimalPerf.default.html) | [604](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/TreeMinimalPerf.default_base.html) | 1.1:1
ProviderMergeThemesPerf.default | [1029](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ProviderMergeThemesPerf.default.html) | [948](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ProviderMergeThemesPerf.default_base.html) | 1.09:1
FormMinimalPerf.default | [291](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/FormMinimalPerf.default.html) | [270](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/FormMinimalPerf.default_base.html) | 1.08:1
TextMinimalPerf.default | [269](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/TextMinimalPerf.default.html) | [249](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/TextMinimalPerf.default_base.html) | 1.08:1
AttachmentMinimalPerf.default | [133](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/AttachmentMinimalPerf.default.html) | [124](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/AttachmentMinimalPerf.default_base.html) | 1.07:1
ListNestedPerf.default | [424](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ListNestedPerf.default.html) | [403](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ListNestedPerf.default_base.html) | 1.05:1
ChatWithPopoverPerf.default | [266](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ChatWithPopoverPerf.default.html) | [257](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ChatWithPopoverPerf.default_base.html) | 1.04:1
DropdownMinimalPerf.default | [2089](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/DropdownMinimalPerf.default.html) | [2032](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/DropdownMinimalPerf.default_base.html) | 1.03:1
GridMinimalPerf.default | [250](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/GridMinimalPerf.default.html) | [243](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/GridMinimalPerf.default_base.html) | 1.03:1
ProviderMinimalPerf.default | [318](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ProviderMinimalPerf.default.html) | [309](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ProviderMinimalPerf.default_base.html) | 1.03:1
CardMinimalPerf.default | [389](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/CardMinimalPerf.default.html) | [380](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/CardMinimalPerf.default_base.html) | 1.02:1
LayoutMinimalPerf.default | [248](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/LayoutMinimalPerf.default.html) | [244](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/LayoutMinimalPerf.default_base.html) | 1.02:1
SkeletonMinimalPerf.default | [297](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/SkeletonMinimalPerf.default.html) | [292](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/SkeletonMinimalPerf.default_base.html) | 1.02:1
StatusMinimalPerf.default | [485](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/StatusMinimalPerf.default.html) | [477](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/StatusMinimalPerf.default_base.html) | 1.02:1
VideoMinimalPerf.default | [556](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/VideoMinimalPerf.default.html) | [546](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/VideoMinimalPerf.default_base.html) | 1.02:1
AvatarMinimalPerf.default | [161](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/AvatarMinimalPerf.default.html) | [160](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/AvatarMinimalPerf.default_base.html) | 1.01:1
CheckboxMinimalPerf.default | [1526](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/CheckboxMinimalPerf.default.html) | [1518](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/CheckboxMinimalPerf.default_base.html) | 1.01:1
DatepickerMinimalPerf.default | [4344](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/DatepickerMinimalPerf.default.html) | [4283](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/DatepickerMinimalPerf.default_base.html) | 1.01:1
ListMinimalPerf.default | [381](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ListMinimalPerf.default.html) | [379](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ListMinimalPerf.default_base.html) | 1.01:1
CustomToolbarPrototype.default | [2136](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/CustomToolbarPrototype.default.html) | [2119](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/CustomToolbarPrototype.default_base.html) | 1.01:1
ListCommonPerf.default | [458](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ListCommonPerf.default.html) | [459](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ListCommonPerf.default_base.html) | 1:1
MenuButtonMinimalPerf.default | [1264](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/MenuButtonMinimalPerf.default.html) | [1262](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/MenuButtonMinimalPerf.default_base.html) | 1:1
SplitButtonMinimalPerf.default | [3346](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/SplitButtonMinimalPerf.default.html) | [3387](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/SplitButtonMinimalPerf.default_base.html) | 0.99:1
ToolbarMinimalPerf.default | [670](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ToolbarMinimalPerf.default.html) | [679](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ToolbarMinimalPerf.default_base.html) | 0.99:1
CarouselMinimalPerf.default | [376](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/CarouselMinimalPerf.default.html) | [383](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/CarouselMinimalPerf.default_base.html) | 0.98:1
EmbedMinimalPerf.default | [2784](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/EmbedMinimalPerf.default.html) | [2844](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/EmbedMinimalPerf.default_base.html) | 0.98:1
TableManyItemsPerf.default | [1453](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/TableManyItemsPerf.default.html) | [1478](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/TableManyItemsPerf.default_base.html) | 0.98:1
ChatMinimalPerf.default | [601](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ChatMinimalPerf.default.html) | [620](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ChatMinimalPerf.default_base.html) | 0.97:1
SliderMinimalPerf.default | [1261](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/SliderMinimalPerf.default.html) | [1304](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/SliderMinimalPerf.default_base.html) | 0.97:1
ListWith60ListItems.default | [452](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ListWith60ListItems.default.html) | [469](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ListWith60ListItems.default_base.html) | 0.96:1
AlertMinimalPerf.default | [208](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/AlertMinimalPerf.default.html) | [218](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/AlertMinimalPerf.default_base.html) | 0.95:1
AttachmentSlotsPerf.default | [835](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/AttachmentSlotsPerf.default.html) | [880](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/AttachmentSlotsPerf.default_base.html) | 0.95:1
LabelMinimalPerf.default | [312](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/LabelMinimalPerf.default.html) | [327](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/LabelMinimalPerf.default_base.html) | 0.95:1
IconMinimalPerf.default | [473](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/IconMinimalPerf.default.html) | [496](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/IconMinimalPerf.default_base.html) | 0.95:1
TooltipMinimalPerf.default | [1717](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/TooltipMinimalPerf.default.html) | [1814](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/TooltipMinimalPerf.default_base.html) | 0.95:1
ItemLayoutMinimalPerf.default | [846](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ItemLayoutMinimalPerf.default.html) | [896](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ItemLayoutMinimalPerf.default_base.html) | 0.94:1
MenuMinimalPerf.default | [605](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/MenuMinimalPerf.default.html) | [646](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/MenuMinimalPerf.default_base.html) | 0.94:1
DialogMinimalPerf.default | [560](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/DialogMinimalPerf.default.html) | [601](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/DialogMinimalPerf.default_base.html) | 0.93:1
ButtonMinimalPerf.default | [127](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ButtonMinimalPerf.default.html) | [140](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ButtonMinimalPerf.default_base.html) | 0.91:1
RefMinimalPerf.default | [161](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/RefMinimalPerf.default.html) | [177](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/RefMinimalPerf.default_base.html) | 0.91:1
RosterPerf.default | [1570](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/RosterPerf.default.html) | [1753](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/RosterPerf.default_base.html) | 0.9:1
TableMinimalPerf.default | [295](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/TableMinimalPerf.default.html) | [330](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/TableMinimalPerf.default_base.html) | 0.89:1
ButtonOverridesMissPerf.default | [955](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ButtonOverridesMissPerf.default.html) | [1080](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ButtonOverridesMissPerf.default_base.html) | 0.88:1
DividerMinimalPerf.default | [255](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/DividerMinimalPerf.default.html) | [294](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/DividerMinimalPerf.default_base.html) | 0.87:1
ImageMinimalPerf.default | [288](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ImageMinimalPerf.default.html) | [336](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/ImageMinimalPerf.default_base.html) | 0.86:1
LoaderMinimalPerf.default | [477](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/LoaderMinimalPerf.default.html) | [554](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/LoaderMinimalPerf.default_base.html) | 0.86:1
FlexMinimalPerf.default | [211](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/FlexMinimalPerf.default.html) | [247](https://fluentuipr.z22.web.core.windows.net/pull/25351/perf-test-northstar/FlexMinimalPerf.default_base.html) | 0.85:1

</details><div id="perfComment9424"/>

File: packages/fluentui/react-northstar/src/themes/teams-dark-v2/siteVariables.ts [hunk 2]
```diff
@@ -148,3 +148,9 @@ export const colorScheme = {
     backgroundDisabled1: colors.grey['800'],
   },
 };
+
+// SHADOW LEVELS
+export const shadowLevel1 = '0 0 2px rgba(0,0,0,0.30), 0 2px 4px rgba(0,0,0,0.25)';
```
Comment (ling1726) [inline packages/fluentui/react-northstar/src/themes/teams-dark-v2/siteVariables.ts:153]: nothing is using these shadows, is that intentional? are updates planned in a future PR?

**Pull request closed without merging**